### ISSUE TITLE
RFTools, missing a block (Elevator Block)

### ISSUE THREAD

**Issue by darkfang87** [AllTheMods/ATM-5#258]
## Expected Behavior

Can't give this part cause the issue is the block is missing, referring to the rftools elevator block
## Current Behavior

Can't give this part cause the issue is the block is missing, referring to the rftools elevator block
## Possible Solution

the elevator block needs to be added in
## Steps to Reproduce

1.
2.
3.
4.
## Detailed Description

All the other parts of the RFtools elevator is there it is missing the elevator block...

**Comment by XenoticK** [comment]
It is not a mistake,the block is intentionally not there. Just go ahead and use PNC - Repressurized elevators if you can until the mod author adds the elevators back! I know they are cool because i love em too but we need to wait a bit for them. Here's the article if you want to check it out: https://github.com/McJtyMods/RFToolsUtility/issues/38

**Comment by darkfang87** [comment]
ok, will wait till then I am using mod called "Moving Elevators", before you ask I play single-player...

**Comment by XenoticK** [comment]
Yeah those are very good,i personally like elevators like RFTools elevator or PNC-repressurized bc they actually use physics at some point!

**Comment by ArchAngel727** [comment]
Any news?

**Comment by rockett1** [comment]
> 
> 
> Any news?

https://www.curseforge.com/minecraft/mc-mods/rftools-utility mod is in alpha and still being worked on.
